netneurotools/networks.py

```diff
@@ -71,8 +83,13 @@ def func_consensus(data, n_boot=1000, ci=95, seed=None):
 
     # generate `n_boot` bootstrap correlation matrices by sampling `t` time
     # points from the concatenated time series
+    if type(data) is list:
+        nsample = np.mean([f.shape[1] for f in data]).astype(int)
```

Comment:
Since we have the stacked data (`collapsed_data`) and the number of subjects (`len(data)`) we can avoid this extra for-loop and do:

```python
if isinstance(data, list):
    nsample = int(collapsed_data.shape[-1] / len(data))
```